docs/.custom_wordlist.txt

```diff
@@ -102,6 +103,7 @@ Matic
 meetingology
 mentorship
 metapackage
+microreleases
```

Comment:
I think we should remove this - if we're defining *micro-releases* in the glossary, it doesn't make sense to add the incorrect term to the exception list

Reply:
Hmm, I only see MRE (Micro Release Exception) in the glossary, and that points to SRU docs, which use 'microrelease'. I don't mind either way, but the non-hyphenated versions seems used more often.

Reply:
> Whichever way we go, I think we should standardise on one and only use that everywhere.

+1 Definitely

As I said, I don't feel strongly about either, but given the current prevalence of non-hyphenated, we might have an easier time with it (not having to fix existing occurrences and not having fight engineers on it).